runtime/doc/autocmd.txt

```diff
@@ -722,6 +724,26 @@ InsertCharPre			When a character is typed in Insert mode,
 				It is not allowed to change the text |textlock|.
 				The event is not triggered when 'paste' is
 				set.
+							*TextYankPost*
+TextYankPost			Just after a |yank| or |deleting| command
+				is issued. Not issued if the black hole register
+				(|quote_|) is used. Information about the yank
+				can be found in the |v:event| dictionary,
+				which has the following keys:
+				regcontents
+					yanked text stored in register as a list
+					of lines.
+				regtype
+					resulting type of register, as returned
+					by |getregtype()|
+				regname
+					requested register ["x] or the empty
+					string for an unnamed operation
+				operator
+					The operation performed ([d]elete,
+					[y]ank, or [c]hange). Unlike |v:operator|,
+					it is correct even if an Ex mode
+					operation was used, like |:yank|
```

Comment:
The documentation of the various keys on `v:event` should probably live at `v:event`. And the autocmds themselves will just mention which keys it actually uses. 

For all autocmds, if the same v:event key is used, it should have the _same_ semantics across any autocmd that uses that key. E.g. `v:event.operator` means the same thing for any event that sets it.

For an autocmd that does not set `v:event.operator`, if a plugin tries `v:event.operator` it will get a `E716: Key not present in Dictionary: v:event`.